assets/sitedata/menus.json

```diff
@@ -105,6 +101,19 @@
 				}
 			]
 		},
+		{
+			"title": "CONTENT",

```

Comment:
I guess "COMMUNITY" would also work. Let me know if you'd like it changed.

Reply:
You are aware it is the fifth option down in the About menu? Just want to put it under projects for more visibility?

Reply:
Hmmm. Just trying to get it more visible because I and others, totally missed it in about :wink:

But maybe having both of them toward the bottom of the Projects menu is better. I'll adjust it that way.